hubble.ipynb

```diff
@@ -0,0 +1,177 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "#Getting each parameters\n",
+    "distance = data.distance.values\n",
+    "recession_velocity = data.recession_velocity.values\n",
+    "\n",
+    "#Setting up the graphs\n",
+    "plt.style.use('seaborn-whitegrid')\n",
+    "fig = plt.figure()\n",
+    "ax = plt.axes()\n",
+    "plt.rcParams.update({'font.size': 12})\n",
+    "plt.title(\"Velocity / Distance\")\n",
+    "ax = ax.set(xlabel='Distance', ylabel='Recession Velocity')\n",
+    "\n",
+    "#Plotting our data\n",
+    "plt.scatter(distance, recession_velocity, label = \"Values\");\n",
+    "\n",
+    "#Reshaping parameters to use LinearRegression\n",
+    "d = distance.reshape(-1, 1)\n",
+    "r = recession_velocity.reshape(-1, 1)\n",
+    "\n",
+    "#Calculating Linear Regression\n",
+    "reg = LinearRegression().fit(d,r)\n",
+    "pred = reg.predict(d)\n",
+    "\n",
+    "#Plotting Linear Regression\n",
+    "plt.plot(distance,pred, color = \"r\", label = \"Regression\")\n",
+    "\n",
+    "plt.legend(loc = 'upper left');\n",
+    "\n",
+    "print(f\"On a une regression donnant la relation :\\n\\nVelocity = {reg.coef_[0,0]} * Distance + ({reg.intercept_[0]})\")"
```

Comment:
If you want to print a float with less decimals you can use `f'Float with 3 decimals: {my_float:.3f}'`